src/pages/Dashboard/Dashboard.tsx

```diff
@@ -6,10 +6,13 @@ import ServerCreationForm from "../../components/ServerCreationForm/ServerCreati
 import ServerSearching from '../../components/ServerSearching/ServerSearching';
 
 import "./Dashboard.scss";
+import { useSelector } from "react-redux";
 
 export default function Dashboard() {
     const [dashboardContent, setDashboardContent] = useState<string>('');
+    const usersState = useSelector((state:any) => state.users);
```

Comment:
Il devrait y avoir moyen de typer le state autrement qu'avec any ici non ?

Reply:
Oui, il y a un moyen, apparemment.
Faut que je potasse la doc redux...

Reply:
Ces deux pages devraient pouvoir te donner une piste je pense:
- https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-state-type
- https://redux.js.org/usage/usage-with-typescript#define-typed-hooks

Reply:
Oui, je les ai mises de côté ^^

Reply:
En ayant analysé le doc, j'ai réalisé qu'on avait déjà fait tout le boulot, l'interface existe déjà.
Done.